### PR TITLE
Added Shogi(Japanese Chess)

### PR DESCRIPTION
**Pull request by Arcturus22**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by Arcturus22** [comment]
@edm00se 
Please review the code and merge it.
The unsuccessful check is due to some link which has been added by someone other contributor. Please resolve the problem or explain what to do.

**Comment by edm00se** [comment]
@Arcturus22 it looks like a false negative on that check, I was able to trigger a re-run which succeeded.

File: readme.md [hunk 1]
```diff
@@ -456,6 +456,16 @@
 | ------: | -------: | ---: |
 |   2 - 8 |        8 |  15m |
 
+### [Shogi](https://en.wikipedia.org/wiki/Shogi)
+
+> Shogi, also known as Game of Generals, is a Japanese board game similar to chess, played on a 9x9 grid with 20 pieces per player. The objective is to checkmate the opponent's king, but pieces can be promoted upon reaching the enemy's promotion zone. Shogi offers a dynamic and strategic gameplay experience with unique piece movements and captures. It is much more complex than international chess. 
+
+![Shogi image](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Shogi_Ban_Koma.jpg/495px-Shogi_Ban_Koma.jpg)
```
Comment (edm00se) [inline readme.md:463]:
```suggestion
![Shogi image](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Shogi_Ban_Koma.jpg/250px-Shogi_Ban_Koma.jpg)
```

Adjusting down to 250px to be consistent with the target image size.

**Comment by Arcturus22** [comment]
> @Arcturus22 it looks like a false negative on that check, I was able to trigger a re-run which succeeded.

Alright, thanks for that!

File: readme.md [hunk 2]
```diff
@@ -456,6 +456,16 @@
...
 
+### [Shogi](https://en.wikipedia.org/wiki/Shogi)
+
+> Shogi, also known as Game of Generals, is a Japanese board game similar to chess, played on a 9x9 grid with 20 pieces per player. The objective is to checkmate the opponent's king, but pieces can be promoted upon reaching the enemy's promotion zone. Shogi offers a dynamic and strategic gameplay experience with unique piece movements and captures. It is much more complex than international chess. 
+
+![Shogi image](https://upload.wikimedia.org/wikipedia/commons/thumb/a/a6/Shogi_Ban_Koma.jpg/250px-Shogi_Ban_Koma.jpg)
+
+| Players | Min. Age |  Time   |
+| ------: | -------: | -----:  |
+|       2 |        5 | 30-120m |
```
Comment (edm00se) [inline readme.md:467]:
```suggestion
|       2 |        8 | 30-120m |
```

Not seeing a minimum age on wikipedia, BGG lists 8.

File: readme.md (hunk 1 above)
Comment (Arcturus22) [reply readme.md]: My bad, I should have done that

**Review by edm00se — APPROVED** [review]
refs #269

**Pull request merged**